x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_winlogbeat_ecs/ml/windows_anomalous_user_name.json

```diff
@@ -0,0 +1,50 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "SIEM Winlogbeat: Detect  users that are not normally active may indicate unauthorized changes or activity by an unauthorized user.",
```

Comment:
Double whitespace chars in here aren't there, after `Detect`? Also needs a `which` after `may`